Tests/ResponseTests.swift

```diff
@@ -672,8 +672,27 @@ final class ResponseTryMapErrorTestCase: BaseTestCase {
         else { XCTFail(); return }
 
         XCTAssertEqual(underlyingError.asAFError?.isSessionTaskError, true)
-        XCTAssertEqual((underlyingError.asAFError?.underlyingError as? URLError)?.code, .cannotFindHost)
-
+        XCTAssertEqual(underlyingError.asAFError?.isHostURLError, true)
         XCTAssertNotNil(response?.metrics)
     }
 }
+
+private extension DataResponse where Failure == AFError {
+    var hasHostURLError: Bool? {
+        return error?.isHostURLError
+    }
+}
+
+private extension AFError {
```

Comment:
You can put this in the `AFError+AlamofireTests.swift` file alongside our other `is*` convenience properties.

Additionally, you can simplify the property down to: 
```swift
return [.cannotConnectToHost, .cannotFindHost].contains((underlyingError as? URLError)?.code)
```